src/java.base/share/classes/java/lang/invoke/InvokerBytecodeGenerator.java

```diff
@@ -85,6 +85,7 @@
 
     /** Name of new class */
     private final String className;
+    private final String prefixedClassName;
```

Comment:
Adding a field for the class name is a good change.   I was tempted to rename `className` at one point to `name` which can be a class name in internal form or a simple name.   It now makes more sense to do the renaming `s/className/name` and `s/prefixedClassName/className`.   What do you think?